### PR TITLE
permissions doc

### PR DESCRIPTION
**Pull request by rkatic**

### CODE REVIEW

File: dev-docs/persmissions-example.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+- Each role can have any number of permissions.
+- Each endpoint can require certain permissions from the user.
+
+```sql
+ALTER TABLE "user"
+  ADD "roles" INTEGER[] NOT NULL DEFAULT '{}',
+
+```
+
+All permission and roles are define as constants in `konst.toml`:
```
Comment (kotarac) [inline dev-docs/persmissions-example.md]: defined

File: dev-docs/persmissions-example.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+  'vendor/account/read',
+  'vendor/account/update',
+  'vendor/account/delete',
+  'vendor/orders/read',
+  'vendor/orders/cancel',
+  'vendor/orders/refund',
+]
+```
+
+All possible permission are listed under `permission.list`.
```
Comment (kotarac) [inline dev-docs/persmissions-example.md]: permissions

File: dev-docs/persmissions-example.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+Each role is stored in DB using respective `val` number. The `can` field holds a "permission selector", and it defines which permissions are included in respective role.
```
Comment (kotarac) [inline dev-docs/persmissions-example.md]: in the respective

File: dev-docs/persmissions-example.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+
+```
+
+
+
+
+
+
+
+
```
Comment (kotarac) [inline dev-docs/persmissions-example.md]: trailing LF

File: dev-docs/persmissions-example.md (hunk 1 above)
Comment (PetarIvancevic) [inline dev-docs/persmissions-example.md]: Shouldn't this be:
```
All permissions and roles are defined as constants in `konst.toml`:
```

**Pull request merged**